src/org/usfirst/frc/team2706/robot/Robot.java

```diff
@@ -107,7 +111,9 @@ public void robotInit() {
      * This function is called once each time the robot enters Disabled mode. You can use it to
      * reset any subsystem information you want to clear when the robot is disabled.
      */
-    public void disabledInit() {}
+    public void disabledInit() {
+        blingSystem.clear(); // S.V.P leave this here. 
```

Comment:
In the Bling command, when it is canceled or ended, it should cancel automatically.